.github/workflows/tests.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches: [master]
```

Comment:
Since this github actions commit will only stay in the parachain branch, I think it makes sense to add the parachain branch as well as trigger.